src/main/java/org/jvnet/hudson/update_center/Plugin.java

```diff
@@ -254,22 +254,9 @@ public String getScmHost() {
         return labels;
     }
 
-    /** @return {@code true} if the plugin's wiki page has the "plugin-deprecated" label. */
-    public boolean isDeprecated() {
```

Comment:
Is there a risk of impacting custom update centers using this class as API?
I see some usages of the codebase, but only in forks. So we unlikely care